draft-ietf-tls-tls13.md

```diff
@@ -1483,11 +1486,12 @@ Structure of this message:
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
            opaque legacy_compression_methods<1..2^8-1>;
-           Extension extensions<0..2^16-1>;
+           [[ Extension extensions<8..2^16-1>; ]]
```

Comment:
I don't think inventing new syntax is a good idea here. Adding a comment on the line stating "OPTIONAL in old versions" would be better.